datahub/config.yaml

```diff
@@ -23,3 +23,45 @@ singleuser:
     extraVolumeMounts:
       - name: home
         mountPath: /home/jovyan
+
+admin:
+  users:
+  # infrastructure
```

Comment:
Can you align the '#' with the '-' in the list? :D That's what I meant I think. Sorry!